ui-v2/src/components/flow-runs/activity-chart/activity-chart.tsx

```diff
@@ -0,0 +1,174 @@
+import { FlowRun } from "@/api/flow-runs";
+import { TooltipProvider } from "@/components/ui/tooltip";
+import { Typography } from "@/components/ui/typography";
+import React, { useCallback, useEffect, useMemo } from "react";
+import { FlowRunCell } from "./flowRunCell";
+
+export type FlowRunsBarChartProps = {
+	startWindow?: Date;
+	endWindow?: Date;
+	mini?: boolean;
+	flowRuns: FlowRun[];
+	flowName: string;
+	className?: string;
+};
+
+const FlowRunsBarChart: React.FC<FlowRunsBarChartProps> = ({
```

Comment:
Curious what the benefit is of using `React.FC` rather than just a plain function. My understanding is that `React.FC` isn't necessary and not recommended anymore.